apps/services/bff/src/app/modules/auth/auth.service.ts

```diff
@@ -297,13 +283,7 @@ export class AuthService {
         codeVerifier: loginAttemptData.codeVerifier,
       })
 
-      if (tokenResponse.type === 'error') {
```

Comment:
Not needed since enhancedFetch will throw the error, which will propagate down to the nearest catch handler.